### DISCUSSION TITLE
Patch problem

### DISCUSSION THREAD

**Discussion by FitzroyWang** [Mr-MIBonk/M.I.B._More-Incredible-Bash#124] [Q&A]
I have a problem with the patch.
I have the software MHI2Q_CN_AUG22_P0915 with MU-0915.
![IMG_8128](https://user-images.githubusercontent.com/126141239/220834951-be60cc75-6861-4e0d-b3ee-662f43c95ac5.JPG)
I followed the instructions but when patching, I got the following error at the end:
![IMG_8096](https://user-images.githubusercontent.com/126141239/220834914-779a551a-fb1f-444d-bcac-f64d0f5797c2.jpg)
NO Flashfiles there! I have checked some times, and I find the FW of patches is right, but the patch file name is a little different，
Search File is MU0915......-30DF7F200.ifs, but patch file name is MU0915......-24DF7F200.ifs
![IMG_8127](https://user-images.githubusercontent.com/126141239/220835006-20d60882-a2d2-4bfd-abec-0c084e16abb5.JPG)
Can I rename this file name? or other solutions？
Sorry for my English, I'm translating from Chinese to English.
Many Thanks for answers.

**Comment by harman-f** [comment]
Could it be, that your unit is already patched? 
please Share log file from SD

**Reply by FitzroyWang** [reply]
this file？
[MU0915-AW7-MON13.02.1800061957.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/10824187/MU0915-AW7-MON13.02.1800061957.log)

**Reply by harman-f** [reply]
ifs-root-stage2 header checksum does not match the information M.I.B has on file.
Therefore, your unit is most likely already patched.

You could rename the file to what M.I.B is looking for.
But this is on your own risk.

Could you share your M.I.B backup from SD card?

**Reply by FitzroyWang** [reply]
Does it mean that I have obtained root permission, I really can modify some content, like Activate dual navigation.

and share this folder?
![a528a344c9e30137166e6e2521392dd](https://user-images.githubusercontent.com/126141239/221341168-fb726968-c200-4523-896b-5767afa4515e.png)

**Reply by harman-f** [reply]
yes, that is the folder.

M.I.B in GEM has root permission and can change stuff, yes.

**Reply by FitzroyWang** [reply]
this folder too big, dont let me upload it.

**Reply by harman-f** [reply]
zip and use filehorst.de

**Reply by FitzroyWang** [reply]
https://filehorst.de/d/emfjCiDz pls download it, it's my first time backup folder.

**Reply by harman-f** [reply]
You already have patched your ifs-root-stage2.ifs.
![image](https://user-images.githubusercontent.com/98130152/221414769-8a89394e-9f5c-4dd8-9ca1-943ef2786c24.png)

**Reply by FitzroyWang** [reply]
Okay, thanks a lot
